xarray/core/rolling_exp.py

```diff
@@ -110,11 +97,16 @@ def __init__(
         obj: T_DataWithCoords,
         windows: Mapping[Any, int | float],
         window_type: str = "span",
+        min_weight: float = 0.0,
     ):
+        if has_numbagg is False or has_numbagg < "0.3.1":
+            raise ImportError("numbagg >= 0.3.1 is required for rolling_exp")
```

Comment:
You can use 
https://github.com/pydata/xarray/blob/e8be4bbb961f58ba733852c998f2863f3ff644b1/xarray/core/pycompat.py#L81-L83
To get the version.

Reply:
Though this isn't a duck array module — numba / numbagg operates on numpy arrays. Would I still use this?